src/platform/silabs/wifi/WifiInterface.h

```diff
@@ -53,6 +53,8 @@ constexpr size_t kWifiMacAddressLength = 6;
 #define WFX_MAX_SSID_LENGTH (32)
 #define MAX_JOIN_RETRIES_COUNT (5)
 
+using WiFiBandEnum = chip::app::Clusters::NetworkCommissioning::WiFiBandEnum;
```

Comment:
We avoid as much as possible to use `using` in a header file since this will propagate to any file that includes the header. 

We should use the long name here.